index.html

```diff
@@ -274,10 +274,10 @@ <h3>Multi-Post Stories</h3>
                             <div class="tools">
                                 <div class="tool-skill">
                                     <ul>
-                                        <li><a href=""></a>css</li>
-                                        <li><a href=""></a>html</li>
-                                        <li><a href=""></a>bootstrap</li>
-                                        <li><a href=""></a>Ruby</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>css</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>html</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>bootstrap</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>Ruby</li>
                                     </ul>
                                 </div>
 
```

Comment:
- [ ] For the contact us section, the section title being 'Contact Us', ensure to main the hierachy in the header tags. Change this to be within `h2` tags.